sequencer/src/persistence/sql.rs

```diff
@@ -90,6 +90,10 @@ pub struct Options {
 
     #[clap(long, env = "ESPRESSO_SEQUENCER_STORE_UNDECIDED_STATE", hide = true)]
     pub(crate) store_undecided_state: bool,
+
+    // Specifies the maximum number of concurrent fetch requests allowed from peers.
+    #[clap(long, env = "ESPRESSO_SEQUENCER_FETCH_RATE_LIMIT", hide = true)]
```

Comment:
Let's not hide this, this is a useful thing that we very much want other operators to use

Reply:
oops I copied the above field :D